src/screens/sync/Devices.tsx

```diff
@@ -175,6 +177,17 @@ const DeviceList = () => {
           {t(m.lastSynced)}
         </Text>
       </View>
+      {[1, 3, 2, 5, 262, 26, 22, 34, 27, 235, 98, 45].map((val) => (
+        <ProgressBar
```

Comment:
each row needs vertical spacing to create visual separation, based on mockup:

mockup:

![image](https://user-images.githubusercontent.com/18542095/230929314-a8fd53f9-c73b-4f7d-953e-924f714aeb48.png)

currently:

<img width="560" alt="image" src="https://user-images.githubusercontent.com/18542095/230929557-3c3bd802-920d-4f38-b680-634cd4d1164c.png">

Reply:
Do you think this should come from the parent? Or should I just hardcode it into the `progress` component?

Reply:
I would usually wrap it in the parent, that way in theory the component can be used elsewhere and let its parent always determine the wrapping spacing for it

Reply:
https://github.com/digidem/mapeo-mobile-shell/pull/23/commits/f4b69366e739e2aab7cdeddeda9ec9cc12745cb4